src/lib/y2storage/btrfs_qgroup.rb

```diff
@@ -0,0 +1,115 @@
+# Copyright (c) [2020] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "y2storage/storage_class_wrapper"
+require "y2storage/disk_size"
+
+module Y2Storage
+  # A qgroup of a Btrfs filesystem
+  #
+  # This is a wrapper for Storage::BtrfsQgroup.
+  #
+  # In Btrfs, the qgroups hold the quota information for a given set of subvolumes.
+  # Although they can be organized in a tree hierarchy, we are only considering the
+  # first level (level 0, actually), so they are associated to a single subvolume.
+  #
+  # @see man btrfs-qgroup
```

Comment:
Please add `@see man btrfs-quota` which contains some more high-level documentation to make sense of what this is all about.

I looked at `man btrfs-qgroup`, and it's a particular useless piece of documentation, written for those who already know everything (like so much of the Btrfs documentation).

Reply:
:+1: